docs/source/set_dev_env_with_vagrant.rst

```diff
@@ -8,15 +8,31 @@ development environment, all you need is just to run
 
     vagrant up --provider virtualbox
 
+Note if you're under proxy, then you need to modify Vagrantfile first by adding proxy
+
+::
+
+    config.proxy.http = "http://;proxy>:<port>"
+    config.proxy.https = "http://<proxy>:<port>"
```

Comment:
Launching the Vagrant and running Nitrate do not require to configure proxy. And the Vagrant way is not a way to run Nitrate as a server, only for local development. This proxy configuration is not specific to Nitrate.

On the other hand, if someone really needs to configure proxy to a Vagrant machine, it should belong to Vagrant knowledge rather than Nitrate.

Reply:
I'm just a dumb user, I don't know about Vagrant, I want the tool to be working. I follow the manual and it doesn't work and there are no clues why. You have to specify it in the manual "If you're behind proxy, don't forget to add those lines"